<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2399</width>
    <height>1694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="global_horiziontal" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="left" stretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QGridLayout" name="left_grid">
         <item row="22" column="0">
          <widget class="QLabel" name="l_det_number">
           <property name="text">
            <string>det_number</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="l_gamma">
           <property name="text">
            <string>gamma</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="l_c">
           <property name="text">
            <string>c</string>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="l_det_rot">
           <property name="text">
            <string>det_rot</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="l_alpha">
           <property name="text">
            <string>alpha</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lE_Spacegroup">
           <property name="text">
            <string>P 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="l_hkl2_p">
           <property name="text">
            <string>hkl2_p</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="25" column="0">
          <widget class="QLabel" name="l_omega_offset">
           <property name="text">
            <string>omega offset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="l_cif_file">
           <property name="text">
            <string>CIF File</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="l_b">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QLabel" name="l_sample_rot">
           <property name="text">
            <string>sample_rot</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="l_a">
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="dSB_wavelength">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>4.740000000000000</double>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="l_spacegroup">
           <property name="text">
            <string>Spacegroup</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="2">
          <widget class="QLabel" name="l_vectors">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Vectors in scattering plane</string>
           </property>
          </widget>
         </item>
         <item row="25" column="1">
          <widget class="QDoubleSpinBox" name="dSB_omega_offset">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-720.000000000000000</double>
           </property>
           <property name="maximum">
            <double>720.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="lE_hkl2_p">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QDoubleSpinBox" name="dSB_det_rot">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-30.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <widget class="QCheckBox" name="cB_unique">
           <property name="text">
            <string>show only unique</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pB_load_cif">
           <property name="text">
            <string>load cif</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pB_clear_cif">
           <property name="text">
            <string>clear cif</string>
           </property>
          </widget>
         </item>
         <item row="24" column="0" colspan="2">
          <widget class="QCheckBox" name="cB_fix_omega">
           <property name="text">
            <string>fix omega offset</string>
           </property>
          </widget>
         </item>
         <item row="20" column="0" colspan="2">
          <widget class="QLabel" name="l_idendity">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Identify:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="l_hkl2">
           <property name="text">
            <string>hkl2</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="l_hkl1">
           <property name="text">
            <string>hkl1</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="l_beta">
           <property name="text">
            <string>beta</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="l_wavelength">
           <property name="text">
            <string>Wavelength</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="dSB_a">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="l_kival">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="dSB_b">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="dSB_c">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="l_ki">
           <property name="text">
            <string>k_i</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="lE_hkl2">
           <property name="text">
            <string>1,1,1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="19" column="0" colspan="2">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="17" column="0" colspan="2">
          <widget class="QCheckBox" name="cB_inplane">
           <property name="text">
            <string>show only inplane</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="lE_hkl1">
           <property name="text">
            <string>1,1,0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lE_cif_filename">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="23" column="1">
          <widget class="QDoubleSpinBox" name="dSB_sample_rot">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QDoubleSpinBox" name="dSB_gamma">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QDoubleSpinBox" name="dSB_alpha">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="dSB_beta">
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="22" column="1">
          <widget class="QSpinBox" name="SB_det_number">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pB">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_ref_list">
          <attribute name="title">
           <string>Reflection List</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableWidget" name="table">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>120</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>60</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>80</number>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>80</number>
             </attribute>
             <column>
              <property name="text">
               <string>h</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>k</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>l</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>q</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>d</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>2theta</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>F2</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>M</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>diff</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>det_rot</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>channel</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>sample_rot</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_powder_simulation">
          <attribute name="title">
           <string>Powder Simulation</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_ps1">
           <item>
            <layout class="QGridLayout" name="gridLayout_ps">
             <item row="4" column="1">
              <spacer name="verticalSpacer_ps">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="l_ps_end">
               <property name="text">
                <string>end</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="l_ps_start">
               <property name="text">
                <string>start</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="l_ps4">
               <property name="text">
                <string>det_rot</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="dSB_powder_start">
               <property name="minimum">
                <double>-20.000000000000000</double>
               </property>
               <property name="maximum">
                <double>-5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDoubleSpinBox" name="dSB_powder_end">
               <property name="minimum">
                <double>-20.000000000000000</double>
               </property>
               <property name="maximum">
                <double>-5.000000000000000</double>
               </property>
               <property name="value">
                <double>-10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="4">
              <widget class="QPushButton" name="pB_ps">
               <property name="text">
                <string>Simulate</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="l_ps_shift">
               <property name="text">
                <string>shift</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="dSB_shift"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="powd_plot_layout"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_sc_simulation">
          <attribute name="title">
           <string>Single Crystal Simulation</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="l_sc_det_rot">
               <property name="text">
                <string>det_rot</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="dSB_sc_det_start">
               <property name="minimum">
                <double>-20.000000000000000</double>
               </property>
               <property name="maximum">
                <double>-5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QLabel" name="l_sc_scample_rot">
               <property name="text">
                <string>sample_rot</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="dSB_sc_sam_start">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
               <property name="value">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="l_sc_sam_start">
               <property name="text">
                <string>start</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QDoubleSpinBox" name="dSB_sc_sam_end">
               <property name="minimum">
                <double>-360.000000000000000</double>
               </property>
               <property name="maximum">
                <double>360.000000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="l_sc_det_end">
               <property name="text">
                <string>end</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="l_sc_sam_end">
               <property name="text">
                <string>end</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="l_sc_det_start">
               <property name="text">
                <string>start</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QDoubleSpinBox" name="dSB_sc_det_end">
               <property name="minimum">
                <double>-20.000000000000000</double>
               </property>
               <property name="maximum">
                <double>-5.000000000000000</double>
               </property>
               <property name="value">
                <double>-10.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="4">
              <widget class="QPushButton" name="pB_sc">
               <property name="text">
                <string>Simulate</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="4">
              <widget class="QLabel" name="l_show_hkl">
               <property name="text">
                <string>hkl =</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0" colspan="4">
              <widget class="QLabel" name="l_warning_ooset">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>159</red>
                     <green>158</green>
                     <blue>158</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Warning: omega offset not set</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="sc_plot_layout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
